Source/Source/iOS/AdsiOSBinding.cs

```diff
@@ -62,11 +64,20 @@ public bool AreInterstitialsEnabled()
             return SMAreInterstitialsEnabled();
         }
 
+        [Obsolete("Use \"void SetHasGDPRConsent(GdprConsent status)\" method instead.")]
         public void SetHasGDPRConsent(bool consent)
         {
-            SMSetHasGDPRConsent(consent);
+            // Using this implementation until iOS native one is in place
+            SetHasGDPRConsent(consent);
+        }
+
+        public void SetHasGDPRConsent(GdprConsent consent)
+        {
+            // Using the above implementation until iOS native one is in place
+            SMSetHasGDPRConsent(consent ? GdprConsent.Granted : GdprConsent.NotGranted);
```

Comment:
be careful if enum are considered ints and start from zero then this code is broken 
   ` 0 ?  0 : 1`

Reply:
OK, will include the cast. Thanks!